remote-config.jsonc

```
@@ -162,7 +165,7 @@
           "message": "Don't forget that `ddev launch` can take a URI, like `ddev launch /admin/reports/status` and don't forget `ddev launch -m` to get the Mailpit UI."
         },
         {
-          "message": "DDEV v1.22.5+ has official support for OrbStack and Rancher Desktop on macOS in addition to Colima and Docker Desktop."
+          "message": "DDEV has official support for OrbStack and Rancher Desktop on macOS in addition to Lima and Docker Desktop. Colima is now discouraged."
```

```suggestion
          "message": "On macOS, DDEV officially supports OrbStack, Rancher Desktop, Lima and Docker Desktop. Colima is now discouraged."
```

i'Ve slightly rearranged the list. it felt odd to have the list of providers cut in half